bftengine/src/bftengine/SimpleClientImp.cpp

```diff
@@ -390,12 +390,12 @@ OperationResult SimpleClientImp::sendRequest(uint8_t flags,
 OperationResult SimpleClientImp::isBatchRequestValid(const ClientRequest& req) {
```

Comment:
I would say logs in isBatchRequestValid could stay as errors because there's an opportunity for earlier pruning of messages that shouldn't have be batched in the first place. These errors could help catch misuse of the batching feature.

Reply:
same as above, a malicious client may send an invalid batch request